Dart/src/com/jetbrains/lang/dart/DartBundle.properties

```diff
@@ -254,6 +254,9 @@ analysis.server.settings.title=Dart Analysis Server Settings
 analysis.server.settings.sdk.settings=Dart plugin settings
 analysis.server.settings.diagnostics=View analyzer diagnostics
 analysis.server.settings.package.analysis=Scope analysis to the current package
+analysis.server.settings.ml.completion=ML Code Completion:
+analysis.server.settings.ml.completion.enable=Enable ML-based autocompletion
```

Comment:
As discussed in email - it'd make sense to have these labels match (or at least be similar) with VS Code. I was trying to avoid using just "ML" since it might not be clear to everyone what that is. Do you have any preferences? (the current text here also uses both "code completion" and "autocompletion", I'm not sure if they should be consistent?)

The current VS Code wording is here:

https://github.com/Dart-Code/Dart-Code/pull/2076

Though re-reading, I'd probably change "machine learning" to "machine learned", since if I understand correctly we're training the model and shipping it, rather than learning on the users machine?

(I don't feel at all strongly about this though, so I'm happy to update it to match whatever you go with here!)

Reply:
The name for this (at tech preview) was `ML Complete`. The abbreviation 'ML' is I think well enough understood now that we can use it in a label (through  tooltip text or help text may use the full text).

As bikeshedding for the labels, perhaps `'ML Completion:'` and `'Enabled ML based code completion'`, or, `'Enabled machine learning based code completion'`?

Reply:
I had an additional reason for not calling it "ML" in the title for VS Code... VS Code auto-generates the display labels from the settings name and would write "ML" as "Ml" 😔 (I've opened https://github.com/microsoft/vscode/issues/83663).

I'll copy whatever is done here though, with just that change (and any tweaks to make it consistent with how other settings appear in VS Code, such as being prefixed with "controls whether...").